portfolio/src/main/webapp/script.js

```diff
@@ -61,7 +61,22 @@ function createCommentElement(comment) {
   const textElement = document.createElement('span');
   textElement.innerText = comment.text;
 
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerText = 'Delete';
+  deleteButtonElement.addEventListener('click', () => {
+    deleteComment(comment);
+
+    // Remove the comment from the DOM.
+    commentElement.remove();
```

Comment:
Pretty cool approach deleting both on the client side and on the server side asynchronously to save time.